standard/clause_6_informative_text.adoc

```diff
@@ -944,6 +955,149 @@ this:location_variable a bald:Array ;
 
 ----
 
+==== Worked Example - Climate and Forecasting Conventions
+
+Variable to variable referencing is a pattern used extensively within the Climate and Forecatsing conventions for netCDF files (CF-netCDF).
+
+CF-netCDF attributes such as `coordinates` can be defined within a specification graph as variable reference attributes, i.e.:
+
+----
+cf:coordinates a owl:ObjectProperty ;
+    rdfs:range bald:Resource .
+----
+
+Given an the specification graph definition and an input CF-netCDF file:
+
+
+----
+netcdf orca2_votemper {
+dimensions:
+        dim0 = 148 ;
+        dim1 = 180 ;
+        time = 4 ;
+        deptht = 4 ;
+
+variables:
+        float votemper(time, deptht, dim0, dim1) ;
+                votemper:_FillValue = 9.96921e+36f ;
+                votemper:standard_name = "sea_water_potential_temperature" ;
+                votemper:long_name = "Temperature" ;
+                votemper:units = "degC" ;
+                votemper:coordinates = "deptht nav_lat nav_lon time" ;
+        float deptht(deptht) ;
+                deptht:units = "m" ;
+                deptht:standard_name = "depth" ;
+                deptht:long_name = "Vertical T levels" ;
+                deptht:positive = "down" ;
+                deptht:title = "deptht" ;
+        float nav_lat(dim0, dim1) ;
+                nav_lat:units = "degrees" ;
+                nav_lat:standard_name = "latitude" ;
+                nav_lat:long_name = "Latitude" ;
+                nav_lat:nav_model = "Default grid" ;
+        float nav_lon(dim0, dim1) ;
+                nav_lon:units = "degrees" ;
+                nav_lon:standard_name = "longitude" ;
+                nav_lon:long_name = "Longitude" ;
+                nav_lon:nav_model = "Default grid" ;
+        int time(time) ;
+                time:units = "hours since 2001-01-01 00:00:00" ;
+                time:standard_name = "time" ;
+                time:long_name = "Time axis" ;
+                time:calendar = "360_day" ;
+                time:time_origin = "2001-JAN-01 00:00:00" ;
+                time:title = "Time" ;
+// global attributes:
+                :Conventions = "CF-1.5" ;
+----
+
+then the `coordinates` attribute defined on the votemper variable is interpreted as a varaible to variable reference, creating 4 explicit reference statements with blank nodes representing the bald:Reference instances.
```

Comment:
"the votemper variable is interpreted as a varaible to variable" --> 
"the votemper variable is interpreted as a variable to variable"

Reply:
typo fixed